src/components/toolbar/ny_toolbar.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import Paginering from './paginering/paginering';
+import Listevisning from './listevisning/listevisning';
+import {ListevisningType} from '../../ducks/ui/listevisning';
+import './toolbar.less';
+import '../../ny_style.less'
+import {useSelector} from 'react-redux';
+import LeggTilArbeidsliste from './legg-til-arbeidsliste-knapp';
+import {AppState} from '../../reducer';
+import ToolbarKnapp from './toolbar-knapp';
+import {ReactComponent as TildelVeilederIkon} from '../ikoner/person-add-1.svg';
+import {ReactComponent as SokVeilederIkon} from '../ikoner/person-view-1.svg';
+import {Undertittel} from 'nav-frontend-typografi';
+import classNames from "classnames";
+import {useWindowWidth} from "../../hooks/use-window-width";
+
+interface ToolbarProps {
+    filtergruppe: ListevisningType;
+    onPaginering?: (fra?: number, antall?: number) => void;
+    sokVeilederSkalVises?: boolean;
+    visesAnnenVeiledersPortefolje?: boolean;
+    children?: React.ReactNode;
+    gjeldendeVeileder?: string;
+    antallTotalt: number;
+    id?: string;
+    antallVeiledere?: number;
+    scrolling?: boolean;
+    isSidebarHidden?: boolean;
+}
+
+function NyToolbar(props: ToolbarProps) {
+    const {id, filtergruppe, visesAnnenVeiledersPortefolje, antallTotalt, onPaginering, scrolling = false, isSidebarHidden = false} = props;
```

Comment:
Har du noen tanker rundt hvilken props du tar med inn i denne variabelen? Ser at det er et par som du senere kaller med props.XXX

Reply:
Vet ikke om jeg skjønner heelt hva du mener? Jeg tar med alle props fra interface ToolbarProps som ligger på linje 17 i samme filen

Reply:
Eller mener du inkonsistensen av bruk av props? :P Isåfall er jeg enig med at det er dårlig, og legger inn en endring med en gang!

Reply:
tenkte på linje 32